### ISSUE TITLE
[BUG] problem

### ISSUE THREAD

**Issue by GalackQSM** [openbullet/openbullet#799]
https://prnt.sc/12vn5xj

**Comment by openbullet** [comment]
What version of OB2 are you using?

**Comment by GalackQSM** [comment]
Version 0.1.14

**Comment by openbullet** [comment]
Could you describe exactly the steps you took in order to produce the error? Were you running jobs? How many? How many bots? (etc.)

**Comment by GalackQSM** [comment]
Well, I just added a config for crack and it gave me that

**Comment by openbullet** [comment]
I'm going to need more information. When did you get the error? Was it when you clicked on the Start button? Can you consistently reproduce it every time you try the same thing?

**Comment by GalackQSM** [comment]
Yes

**Comment by openbullet** [comment]
I tried and cannot reproduce this. Have you tried restarting OB2? Can you try an older build to confirm that the error wasn't introduced in this update?

**Comment by GalackQSM** [comment]
I downloaded the realase on github

**Comment by openbullet** [comment]
Yes, I mean can you try 0.1.13 and see if the error is still present?

**Comment by GalackQSM** [comment]
Yes present
